core/src/main/java/hudson/node_monitors/DiskSpaceMonitor.java

```diff
@@ -78,10 +90,13 @@ protected Callable<DiskSpace, IOException> createCallable(Computer c) {
 
             return p.asCallableWith(new GetUsableSpace());
         }
-    };
+    }
 
-    @Extension
+    /**
+     * @deprecated since TODO
+     */
+    @Deprecated
     public static DiskSpaceMonitorDescriptor install() {
-        return DESCRIPTOR;
+        return (DiskSpaceMonitorDescriptor) Jenkins.get().getDescriptor(DiskSpaceMonitor.class);
     }
```

Comment:
Probably reasonable to delete (after confirming nobody calls it to be safe). This method only exists to instantiate the class for `@Extension`, and based on https://github.com/jenkinsci/jenkins/commit/f592d33a7ecf2c5806ae41c827f5d93b17da87b2#diff-61ebef87257e01fc5f2b0a6c82d54c638748cb3e10e5f476170e03124356bc91R102-R106 only because it used to be conditionally instantiated, with https://github.com/jenkinsci/jenkins/commit/4825967fe322bd84bf968773d37e4655366120b9#diff-61ebef87257e01fc5f2b0a6c82d54c638748cb3e10e5f476170e03124356bc91R78 having been incomplete cleanup.

Reply:
Seems the method is not used anywhere. The same method exists for TemporarySpaceMonitor, so lets delete them both